app/components/GetStarted.js

```diff
@@ -254,6 +255,7 @@ class Home extends Component{
           <Step>
             <StepLabel>Start RPC Concensus</StepLabel>
             <StepContent>
+              <ShowError error={startRpcError} />
               {}
```

Comment:
Can you define a new const for 
startRpc = (
<div>
  <ShowError>
</div>
);
then in put the startRpc in the curly brackets there.

Basically the same way it is done for the other steps